CHANGES.rst

```diff
@@ -1175,6 +1175,9 @@ astropy.units
 - Ensure ``keepdims`` works for taking ``mean``, ``std``, and ``var`` of
   ``Quantity``. [#11198]
 
+- Use proper precision when creating a string from a ``Quantity`` in case precision format is not
```

Comment:
Sorry to have another nitpick, but I think this suggestion of mine doesn't actually convey to a reader what changed. Maybe
```
  - For ``Quantity.to_string()``, ensure that the precision argument is also
    used when the format is not latex. [#11145]
```

Reply:
Fixed in https://github.com/astropy/astropy/commit/179bf320e769dcc6acba921a762bb83cf56f6313 , also squashed this bug fix into one commit.